frontend/src/constants/actionTypes.ts

```diff
@@ -21,6 +21,21 @@ export const DELETE_PARCEL = 'DELETE_PARCEL';
 export const STORE_LOOKUP_CODE_RESULTS = 'STORE_LOOKUP_CODE_RESULTS';
```

Comment:
I really hate these files lol.  Just pointless boiler-plate that don't do anything for us.  It's why I'm leaning heavily into the new redux toolkit.